Makefile

```diff
@@ -0,0 +1,43 @@
+.DEFAULT_GOAL := help
+
+WORKING_DIR := chat
+JS_TARGET := $(WORKING_DIR)/public/js/translations
+EXTRACT_DIR := $(WORKING_DIR)/translations/en/LC_MESSAGES
+EXTRACTED_DJANGO := $(EXTRACT_DIR)/django-partial.po
+EXTRACTED_DJANGOJS := $(EXTRACT_DIR)/djangojs-partial.po
+EXTRACTED_TEXT := $(EXTRACT_DIR)/django.po
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+upgrade:
+	pip-compile --output-file test_requirements.txt test_requirements.in
+
+extract_translations: ## extract strings to be translated, outputting .po files
+	cd $(WORKING_DIR) && i18n_tool extract
+	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
+	tail -n +20 $(EXTRACTED_DJANGOJS) >> $(EXTRACTED_TEXT)
+	rm $(EXTRACTED_DJANGOJS)
+	sed -i'' -e 's/nplurals=INTEGER/nplurals=2/' $(EXTRACTED_TEXT)
+	sed -i'' -e 's/plural=EXPRESSION/plural=\(n != 1\)/' $(EXTRACTED_TEXT)
+
+compile_translations: ## compile translation files, outputting .mo files for each supported language
+	cd $(WORKING_DIR) && i18n_tool generate
+	python manage.py compilejsi18n --namespace ChatXBlockI18N --output $(JS_TARGET)
```

Comment:
This step is failing as well.

Reply:
@viadanna This is not failing for me. Can you paste the output you're seeing?

Reply:
```
ImportError: No module named mako.template
Makefile:26: recipe for target 'compile_translations' failed
```

Sorry, should have been more specific. Just missing `mako` from the requirements.

Reply:
@viadanna Ah yeap, I did that manually but forgot to put it in the requirements. Let me add it.